build.gradle

```diff
@@ -160,6 +160,7 @@ task javaPreCommitPortabilityApi() {
 
 task javaPostCommit() {
   dependsOn ":runners:google-cloud-dataflow-java:postCommit"
+  dependsOn ":runners:google-cloud-dataflow-java:postCommitRunnerV2"
```

Comment:
I like the idea that we add runner v2 post commit into current test suite. One further question I have is that should we consider have dataflow post commit separated from java post commit? cc: @tysonjh